Core/Core/Assets.xcassets/icons/attachment.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This icon is available in https://instructure.design/#icons-svg Can we add `paperclip` to the build-icons script instead of adding it as manual 1-off icon?

Reply:
Heck yeah. I forgot we only pull down the ones we need.